giraffe/src/transforms/line.ts

```diff
@@ -28,13 +51,17 @@ export const lineTransform = (
   for (let i = 0; i < table.length; i++) {
```

Comment:
not a blocker, just a thought for the future: might consider converting this to `forEach` function if at all possible and unless there's a strong reason to use a `for` loop - manually iterating is sooooooooo 1989